test/apps/iodemo/io_demo.cc

```diff
@@ -2135,11 +2112,13 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
             std::cout << "  -P <interval>               Set report printing interval"  << std::endl;
             std::cout << "" << std::endl;
             std::cout << "  -q                          Validate data in buffers" << std::endl;
+            std::cout << "  -m <host>"
 #ifdef HAVE_CUDA
-            std::cout << "  -m <host|cuda|cuda-managed> Memory type to use" << std::endl;
+                      << "|cuda|cuda-managed"
 #else
-            std::cout << "  -m <host>                   Memory type to use" << std::endl;
+                      << "                  "
 #endif
+                      << " Memory type to use" << std::endl;
```

Comment:
this looks worse..
maybe print like this, to avoid else part:
```
-m <memory-type>               Memory type to use. possible values: host, cuda, cuda-managed
```